problems/median_two_sorted_arrays.py

```diff
@@ -0,0 +1,36 @@
+#https://leetcode.com/problems/median-of-two-sorted-arrays/submissions/
+
+from typing import List
+
+class Solution:
```

Comment:
This is a nice brute force solution, but we can do better.

leetcode wants O(log (m+n)). What does that mean?
<details>
  <summary>Hint 1</summary>
  
  It means we need to cut our search space in half every time. 

  Is there a property of the input arrays that is helpful here?
  
</details>

<details>
  <summary>Hint 2</summary>
  
  The input arrays are sorted. 

  how can we find a value in a sorted array quickly?
  
</details>

<details>
  <summary>Hint 3</summary>
  
  A binary search. 
  
  Now how do we use a binary search to find a median between two arrays?
  
</details>

Reply:
The point I want to hit home with this comment is to use time/space complexity bounds to help you narrow what algorithms and data structure are available to you. You then look at those options left to see what can solve your problem. This is a key process in optimization. find the upper bound (brute force) and identify what algorithms and data structures are available in lower bounds that could be applied to your problem space in a novel way. Being given a target big O should be a huge hint that you should never waste.